app/code/Magento/Swatches/Model/SwatchAttributesProvider.php

```diff
@@ -36,10 +40,12 @@ class SwatchAttributesProvider
      */
     public function __construct(
         Configurable $typeConfigurable,
-        SwatchAttributeCodes $swatchAttributeCodes
+        SwatchAttributeCodes $swatchAttributeCodes,
+        \Magento\Swatches\Helper\Data\Proxy $swatchHelper
```

Comment:
Please, set proxy $swatchHelper  in di.xml (remove \Proxy from construct and move it to di.xml)

Proxies and interceptors MUST NEVER be explicitly requested in constructors. See:
http://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html